### ISSUE TITLE
Staging Review finding: Non-unique names for edit buttons on review page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93008]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10d CHAMPVA Application Form
**Feature:** new product

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/apply-form-10-10d/review-and-submit](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/review-and-submit), there are a number of instances in which "Edit" buttons do not have unique accessible names. For screen reader users who don't have the visual context around the page, it's difficult to determine what each button is associated with.

In the "Signer information" section, multiple sections have the same heading, "Certification," which then populates the `aria-label` for the edit button. I count four certification sections, so four "Edit certification" buttons.

In the "Applicant information" section, some sections (but not all) do not have an `aria-label` --- meaning they're announced just as "Edit." I'm seeing that happening for "sex listed at birth," "relationship to sponsor," "Medicare Part A and B status," and "other health insurance."

There are also duplicate headings in "Applicant information" for "[Name]'s other health insurance" for the question and the file uploads --- one section's edit button has an `aria-label`, one doesn't. They don't duplicate each other right now, but they will when the missing `aria-label` is resolved!

<details><summary>Link, screenshot or steps to recreate</summary>

![Review page with multiple "Certification" sections repeated.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/70913-Brian-4.png?raw=true)

![Review page with sections "sex assigned at birth" through "other health insurance."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/70913-Brian-5.png?raw=true)

</details>

## Recommended action

1. Make sure that no pages within a step have the same H3 heading, which then populates on the review page. Making these headings unique should ensure each has a unique `aria-label`.
2. Make sure that no "Edit" buttons are missing their `aria-label`. Each should have an `aria-label` matching their corresponding heading.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jamiefiore** [comment]
Here's the[ Figma file](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=3929-25870&t=sVAcTLcvxP3WqiVD-4) with the suggested changes

**Comment by michaelclement** [comment]
Need to update some e2e tests from other teams that relied on edit button behavior.

**Comment by michaelclement** [comment]
@baltes This has been merged and should be visible on staging shortly. To review, please see the above screenshots. Thank you!
